projection/validator/migrations/20221031152131_alter_view_validators_add_attention.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE view_validators
+    ADD attention BOOL NOT NULL DEFAULT FALSE;
+
+CREATE INDEX view_validators_attention_btree_index ON view_validators USING btree (attention);
```

Comment:
May i know why this flag needs to be indexed?

Reply:
Iriginally, I think we will filter by this flag, but seems like it is not the case. I will remove it~